src/EnergyPlus/GroundTemperatureModeling/GroundTemperatureModelManager.hh

```diff
@@ -74,20 +74,11 @@ namespace GroundTemperatureManager {
 
 struct GroundTemperatureManagerData : BaseGlobalStruct
 {
-    Array1D_string const CurrentModuleObjects = Array1D_string({0, 6},
-                                                               {"Site:GroundTemperature:Undisturbed:KusudaAchenbach",
-                                                                "Site:GroundTemperature:Undisturbed:FiniteDifference",
-                                                                "Site:GroundTemperature:BuildingSurface",
-                                                                "Site:GroundTemperature:Shallow",
-                                                                "Site:GroundTemperature:Deep",
-                                                                "Site:GroundTemperature:FCfactorMethod",
-                                                                "Site:GroundTemperature:Undisturbed:Xing"});
-
     std::vector<std::shared_ptr<BaseGroundTempsModel>> groundTempModels;
 
     void clear_state() override
     {
-        this->groundTempModels.clear();
+        *this = GroundTemperatureManagerData();
```

Comment:
I was looking through the coverage test results and noticed that 1) none of these `clear_state` calls are appearing in the coverage results (though they surely have to be being hit), and 2) this one was inconsistent with the other `clear_state` calls. This fixes up this inconsistency.

Reply:
Actually, we are using this now instead `new (this) GroundTemperatureManagerData();`. It calls the default constructor on `this` directly rather than allocating a new object and then assigning it to `this`.